### DISCUSSION TITLE
UI using Windows 11 icons/theme

### DISCUSSION THREAD

**Discussion by PCUser2021** [setvisible/DownZemAll#184] [Ideas]
Theme: Win 10
Icons: Win 11

Thanks

**Comment by setvisible** [comment]
Hi

Links of references to the icons would be appreciated.

Regards

**Comment by PCUser2021** [comment]
https://dekisoft.com/wp-content/uploads/2021/08/7TSP-Windows-11-Icon-Pack.jpg
https://dekisoft.com/windows-11-icons-pack-installer-download/
